scilpy/utils/metrics_tools.py

```diff
@@ -212,3 +211,31 @@ def plot_metrics_stats(mean, std, title=None, xlabel=None,
 
     plt.close(fig)
     return fig
+
+
+def get_roi_metrics_stats(density_map, metrics_files):
+    """
+    Returns the mean and standard deviation of each metric, using the
+    density map provided. This can be a binary mask,
+    or contains weighted values between 0 and 1.
```

Comment:
english question: using the density map provided or using the provided density map?

This can ... contain weighted values. No s on contain.